Leanplum/Leanplum-iOS-SDK
# LP-10807: Fix for the safe area for bottom padding

What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [LP-10807](https://leanplum.atlassian.net/browse/LP-10807)
People Involved   | @aafrooze , @gg4race , @e7mac 

## Background
Customer reported different UI for Center Popup acceptButton for iPhone x. This was due to safe area insets bottom padding added to all templated, Interstitial fullscreen vs non fullscreen.

## Implementation
Add check if the message is centerpopup, not to add the bottom padding.

## Testing steps
Run on iPhone x and non iPhone x and check centerpopup. There should be no difference.

## Is this change backwards-compatible?

**Before:**
<img width="545" alt="Screen Shot 2019-07-22 at 2 16 07 PM" src="https://user-images.githubusercontent.com/45770277/61816167-aa2fc080-ae00-11e9-8557-7df52a79f00d.png">

**After:**
<img width="545" alt="Screen Shot 2019-07-22 at 2 41 22 PM" src="https://user-images.githubusercontent.com/45770277/61816134-92583c80-ae00-11e9-9750-e07a63e690eb.png">
